### DISCUSSION TITLE
howto disbale input-buffers

### DISCUSSION THREAD

**Discussion by wolfgang6444** [SpenceKonde/megaTinyCore#1185] [Q&A]
The microchip spec for attiny 1616 states under 16.3:

Important: For lowest power consumption, disable the digital input buffer of unused pins and pins that
are used as analog inputs or outputs.

This makes absolute sense to me, but I could not figure out how to achieve this. 
PinxCTRL only talks about interrupts, pull-ups, and inversion - not about disabeling the input buffer.

**Comment by hmeijdam** [comment]
![image](https://github.com/user-attachments/assets/224ea5c4-595a-4e50-b310-626afe4f5388)

**Comment by SpenceKonde** [comment]
We also provide a wrapper around this that efficiently allows setting, clearing and in some cases even toggling, detailed in the digital i/o documentation, called pinConfig()
